tests/accessor/accessor_api_local_common.h

```diff
@@ -97,6 +98,23 @@ class check_local_accessor_api_methods {
         check_return_type<explicit_pointer_t<T, target>>(log, acc.get_pointer(),
                                                          "get_pointer()");
       }
+
+      /** check local accessor type alias
+      */
+      static_assert(std::is_same<
+          decltype( local_accessor<T,dims>{range, h} ),
+          decltype( accessor<T, dims, mode, target> {range, h} )
+          >::value, "local accessor type alias check");
+
+      /** check default value of dims parameter in local accessor type alias
+      */
+      if (1 == dims) {
```

Comment:
`static_assert` is compile time check, so this `if` statement do not enable it only for `1 == dims` case.